test/TEST-35-ISCSI-MULTI/test.sh

```diff
@@ -317,6 +317,7 @@ test_setup() {
         inst_multiple poweroff shutdown
         inst_hook shutdown-emergency 000 ./hard-off.sh
         inst_hook emergency 000 ./hard-off.sh
+        inst_simple ./99-default.link /etc/systemd/network/99-default.link
```

Comment:
Is this still needed with the `modules.d/35network-legacy/module-setup.sh` change?